rskj-core/src/main/resources/expected.conf

```diff
@@ -232,6 +232,9 @@ sync = {
     waitForSync = <bool>
 }
 rpc = {
+    gascap =  <number>
+    timeout = <number>
+    timeoutunit = <string>
```

Comment:
Do you really think we need `timeoutunit` as a configurable value? I can only imagine a timeout that can be represented in seconds. Am I missing anything?